File: SPACE TRIANGLE PATTERN/code10.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);
	int temp = 1;

	for(int i = 1 ; i <= row ; i++){

		for(int a = 0 ; a < row - i ; a++){

			printf(" \t");
		}

		for(int j = 1; j <= i ; j++){

			printf("%d\t",temp);
			if(j != i){

				temp += row;
			}
			
		}

		printf("\n\n");

	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/8/code10.java
import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("enter no. of rows");
		int row = s.nextInt();
		int temp = 1;


		for(int i = 1; i <= row ; i++){
			for(int j = 0;j <row ;j++){
                        	if(i % 3 == 0){ 
				
					System.out.print("@		");
				}else if(i % 2==0){

					System.out.print("$		");

				}else{
					System.out.print("#		");
				}

			
			}
			
			
			System.out.println("\n");
		}
	}
}
